src/fitnesse/slim/SlimSymbol.java

```diff
@@ -0,0 +1,81 @@
+package fitnesse.slim;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+abstract public class SlimSymbol {
+  public static final Pattern SYMBOL_PATTERN = Pattern
+      .compile("\\$([A-Za-z\\p{L}][\\w\\p{L}]*)");
+// This would be a better pattern as it allows to define the end of a symbol name with another $ sign  
```

Comment:
Why leave the outcommented pattern here? If it is a better pattern, why has it not been used? Maybe include the reason for not using it at least if the comment should stay here.